### ISSUE TITLE
Q: merge with another project 

### ISSUE THREAD

**Issue by gulldan** [ivre/masscan#8]
hello, hi
would you mind to merge with https://github.com/bi-zone/masscan-ng ?
they add https and MSF, seems interesting

**Comment by p-l-** [comment]
Thanks for the heads up. Overall I don't think it matches our needs & expectations; for example the commit history is super hard to follow.

**Comment by mzpqnxow** [comment]
@gulldan you can reference this to understand the reasoning behind that response:

https://github.com/ivre/masscan/pull/1#issuecomment-1221070771

Most of the commits to that fork (as far as I can tell) are small changes/refactors that touch many lines of code without adding any _runtime_ value

I love "correct" code but not when it adds cost somewhere else. In this context, those commits only serve to make it more difficult to keep in sync with upstream, which in turn makes it more difficult to cherry-pick features/bug-fixes to/from upstream
